src/random.ts

```diff
@@ -8,5 +8,5 @@ export const randexp = (pattern: string | RegExp, flags?: string) => {
 }
 
 export const seed = (value: number) => {
-  Faker.seed.call(Faker, value)
+  Faker.seed.call(Faker, [value])
```

Comment:
@Hourglasser23 Please revert this. (Fixed in https://github.com/iendeavor/yup-schema-faker/commit/260b9f9802284185ebfe01c018138476f73f992b)

Reply:
Supper you changed your mind, no reverting then :)